pkgs/applications/misc/activate-linux/default.nix

```
@@ -24,19 +27,38 @@ stdenv.mkDerivation rec {
   ];
```

```suggestion
  wayland-scanner
  ];
```

wayland-scanner is used

---

Fixed

---

it's not in the dependencies in the README but it's in the source code I guess.